Test/TestHelper.cs

```diff
@@ -153,5 +154,16 @@ private static void Configuration_ConfigurationFailed(object sender, Configurati
         {
             Assert.Fail(e.Message);
         }
+
+        public static void WebFetch()
```

Comment:
Is this change really necessary anymore?

Reply:
It isn't. We could drop it if we don't expect other tests to need to generate web requests to generate traffic.

Reply:
Not likely